electron-builder.json

```diff
@@ -9,19 +9,7 @@
     "hardenedRuntime": true,
     "entitlements": "./scripts/entitlements.mac.inherit.plist",
     "entitlementsInherit": "./scripts/entitlements.mac.inherit.plist",
-    "type": "distribution",
-    "binaries": [
```

Comment:
These are no longer needed, because the electron-builder scans for & signs dependencies in root `node_modules` by default

Reply:
Looks like `packages/app` is bundling chokidar in an unhoisted node_modules, which is breaking darwin build: 
![image](https://user-images.githubusercontent.com/1151760/142650252-3bca5855-2dcc-4971-ab5a-612ea9709c7a.png)

Reply:
It's not `packages/app`, `app` is the dir we build to, so it's the root. From searching around on other issues, it seems like it's treating `.node` as a hidden file and should be fixed by the latest electron tooling releases - bumping to see if that helps.

Reply:
Alternatively we can just add that back to `binaries":` but it'd be nice to see if it can be automatically handled

Reply:
> It's not `packages/app`, `app` is the dir we build to, so it's the root. 

Oh, blah, that was a pre-coffee comment. :laughing: 

> bumping to see if that helps.

:+1:

Reply:
Nevermind, bumping introduced other issues, just adding back to "binaries" and seeing if it builds